app/views/sessions/new.html.erb

```diff
@@ -8,6 +8,10 @@
         <div class="panel-body">
           <%= form_tag :login, html: { class: 'form-hozisontal' } do %>
             <div class="form-group">
+              <%= label_tag :name %>
```

Comment:
Done..

On Thu, Apr 9, 2015 at 1:16 PM, Ben Woodall <notifications@github.com>
wrote:

> In app/views/sessions/new.html.erb
> <https://github.com/benwoody/shrtnr/pull/4#discussion_r28098319>:
>
> > @@ -8,6 +8,10 @@
> >          <div class="panel-body">
> >            <%= form_tag :login, html: { class: 'form-hozisontal' } do %>
> >              <div class="form-group">
> > +              <%= label_tag :name %>
>
> I see the need for adding a name input when creating a user, but it
> shouldn't be there when we authenticate a user. The field can be blank and
> still work. Keep it at email and password.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/benwoody/shrtnr/pull/4/files#r28098319>.
>



-- 
Joe Simpson
“Reasonable people adapt themselves to the world. Unreasonable people
attempt to adapt the world to themselves. All progress, therefore, depends
on unreasonable people.”
George Bernard Shaw